tests/InputFiles/Burgers/Step.yaml

```diff
@@ -1,10 +1,12 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-# Executable: EvolveBurgers
-# CommandLineArgs: +balancer RandCentLB +p2
-# Timeout: 5
-# Check: parse;execute
+Executable: EvolveBurgers
+CommandLineArgs: +balancer RandCentLB +p2
+Timeout: 5
+Check: parse;execute
```

Comment:
Can we somehow separate the test-related options from the production-related options, maybe in a "Testing" group or something?  Having a "CommandLineArgs" option in the example that is completely ignored in production is confusing.

Reply:
Yes good idea!